docs/version-history/lts-versions-23-0.rst

```diff
@@ -36,6 +36,9 @@ Bugs Fixed:
   transfer plugin.
   :jira:`2273`
 
+- Fixed bug where the Shadow failed to write its job ad to ``JOB_EPOCH_HISTORY``
```

Comment:
For future consideration: would this be better as macro:`JOB_EPOCH_HISTORY`, so that it links to the relevant place?

Reply:
I couldn't remember if the `:macro:` role was available in the LTS series, but it appears it does so I am going to change that